managedkafka/examples/pom.xml

```diff
@@ -40,13 +40,7 @@
     <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-managedkafka</artifactId>
-        <version>0.1.0</version>
```

Comment:
Why was this dependency removed?

Reply:
I was following the guidance that was provided here: https://cloud.google.com/java/docs/reference/google-cloud-managedkafka/latest/overview#maven 

There is no version provided for this dependency and removing the version caused errors importing that specific version of grpc. Moreover, I'm not sure why it was added and removing it didn't affect being able to run the client library.

Talked to Adam and couldn't exactly figure out the reason this was added unless I'm missing something?